simulator/scheduler/plugin/resultstore/store.go

```diff
@@ -185,6 +208,29 @@ func (s *Store) AddFilterResult(namespace, podName, nodeName, pluginName, reason
 	s.results[k].filter[nodeName][pluginName] = reason
 }
 
+// AddPostFilterResult adds post filtering result ot the pod annotaiton.
+// - nominatedNodeName represents the node name which nominated by the postFilter plugin.
+//   Otherwise, the string "unschedulable" would be stored in this arg.
+func (s *Store) AddPostFilterResult(namespace, podName, nominatedNodeName, pluginName string, nodeNames []string) {
+	s.mu.Lock()
+	defer s.mu.Unlock()
+
+	k := newKey(namespace, podName)
+	if _, ok := s.results[k]; !ok {
+		s.results[k] = newData()
+	}
+	for _, nodeName := range nodeNames {
+		if _, ok := s.results[k].postFilter[nodeName]; !ok {
+			s.results[k].postFilter[nodeName] = map[string]string{}
+		}
+		if nodeName == nominatedNodeName {
+			s.results[k].postFilter[nodeName][pluginName] = PostFilterNominatedMessage
+		} else {
+			s.results[k].postFilter[nodeName][pluginName] = PostFilterUnschedulableMessage
```

Comment:
The AddPostFilterResult stores whether the node is nominated or not.
Is this enough?

Reply:
One information good to add is who the scheduling Pod kills. it'll increase the complexity of implementation though... WDYT?

(Anyway, if we decide to implement it, I think we can do it later in a separate PR after this PR get merged.)

Reply:
> One information good to add is who the scheduling Pod kills.

We can implement it by watching delete Pods events because only one PostFilter plugin can be registered in scheduler. 
Yeah, If, by bad luck, another Pod on a nominated Node is deleted at the same time PostFilter running, then incorrect information will be recorded.

Reply:
> One information good to add is who the scheduling Pod kills.

yup, I finally understand what the issue is... I think it is needed, too. 
ok, I'll give it some consideration.

Reply:
> only one PostFilter plugin can be registered in scheduler.

BTW, is this misunderstanding?
although just only one postfilter plugin can return the result, we can register it any number, I think.

> postFilter: These plugins are called in their configured order when no feasible nodes were found for the pod. If any postFilter plugin marks the Pod schedulable, the remaining plugins are not called.
https://kubernetes.io/docs/reference/scheduling/config/#extension-points

Reply:
> although just only one postfilter plugin can return the result, we can register it any number, I think.

Ohh, tashikani. I mistakenly understood so far 😅